### PR TITLE
finished task

### PR DESCRIPTION
**Pull request by BGBroth**
by-manufacturers/
image
cars/
image
cars/1/
image
drivers/
image
drivers/3/
image

### CODE REVIEW

File: taxi/urls.py [hunk 1]
```diff
@@ -1,9 +1,29 @@
 from django.urls import path
 
-from .views import index
+from .views import (
```
Comment (viktoria-rybenchuk) [inline taxi/urls.py]: use absolute imports instead of relative

File: taxi/views.py [hunk 2]
```diff
@@ -13,3 +14,38 @@ def index(request):
     }
 
     return render(request, "taxi/index.html", context=context)
+
+
+class ManufacturerListView(generic.ListView):
+    model = Manufacturer
+    queryset = Manufacturer.objects.order_by("name")
+    paginate_by = 5
+    template_name = "taxi/manufacturer_list.html"
```
Comment (viktoria-rybenchuk) [inline taxi/views.py]: Do you really need override template name?

File: taxi/views.py [hunk 3]
```diff
@@ -13,3 +14,38 @@ def index(request):
...
+
+
+class CarListView(generic.ListView):
+    model = Car
+    paginate_by = 5
+    template_name = "taxi/car_list.html"
```
Comment (viktoria-rybenchuk) [inline taxi/views.py]: the same here

Comment (viktoria-rybenchuk) [reply taxi/views.py]: Fix in all places

**Review by viktoria-rybenchuk — CHANGES_REQUESTED** [review]

**Review by pavlejviki — APPROVED** [review]
Good job

**Pull request closed without merging**